lib/Makefile.vc

```diff
@@ -2,22 +2,22 @@
 all : libsleef libsleefgnuabi libsleefdft
 
 .PHONY: libsleef
-libsleef : libsleef.dll
+libsleef : sleef.dll
```

Comment:
This changes in the Makefile build system are completely unexpected. Why do you need them? This patch is building libdft with cmake, it should ignore the Makefiles.

Reply:
Probably it is better to make this another pull request, but this is necessary since the cmake is already generating a library under the name sleef.dll. Without changing in this way, sleef.dll cannot be linked with Windows. See the change in sleeflibm.h.org.